src/components/PatchNotes.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+
+// PatchNotes follows structure/layout of AboutPage.tsx
+const PatchNotes = () => {
+    const [isOpen, setIsOpen] = useState(true);
+
+    // show modal only on first visit    
+    useEffect(() => {
+        if (localStorage.getItem('visitedcount')) {
```

Comment:
Another nit but I think having the explicit conditional is better for code maintainability even though this is such a simple property.